questions.md

```diff
@@ -0,0 +1,4 @@
+# Questions regarding pull requests
+
+    * Do we have to always create a pull request on a branch ?
+    * Can we do some action based on the feedback from the pull requests ?
```

Comment:
You can alter and update your code, push to the branch of your PR, and ask for additional feedback from there.